budgetportal/summaries.py

```diff
@@ -553,3 +557,55 @@ def get_aggregate_drilldowns(self):
             openspending_api.get_programme_name_ref(),
             openspending_api.get_econ_class_4_ref(),
         ]
+
+
+class BudgetedAndActualComparison(DepartmentBudgetData):
+    def __init__(self, department, financial_year):
+        super().__init__(department)
+        self.financial_year = financial_year
```

Comment:
any reason why you can't just use `department.government.sphere.financial_year` and avoid overriding this?

Reply:
done